File: jiaoyu-teacher-web/src/main/webapp/js/service/mynewsService.js
//服务层
app.service('mynewsService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../mynews/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../mynews/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../mynews/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../mynews/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../mynews/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../mynews/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../mynews/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: jiaoyu-teacher-interface/src/main/java/com/jiaoyu/teacher/service/MynewsService.java
package com.jiaoyu.teacher.service;
import java.util.List;
import com.jiaoyu.pojo.Mynews;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface MynewsService {
	
	
	/**
	 * 按时间排序！！最新的新闻
	 * @return
	 */
	public List<Mynews> findNewNewsList();

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<Mynews> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Mynews mynews);
	
	
	/**
	 * 修改
	 */
	public void update(Mynews mynews);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Mynews findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long [] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Mynews mynews, int pageNum,int pageSize);
	
}
